suite-native/module-device-onboarding/src/components/wallet-backup/WalletBackupCard.tsx

```diff
@@ -0,0 +1,79 @@
+import { memo } from 'react';
+import { Pressable } from 'react-native';
+
+import { Card, CardDivider, FullAlertBox } from '@suite-native/atoms';
+import { FullAlertProps } from '@suite-native/atoms/src/FullAlertBox/types';
+import { useTranslate } from '@suite-native/intl';
+import { useOpenLink } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { HELP_CENTER_MULTI_SHARE_BACKUP_URL } from '@trezor/urls';
+
+import { CardContent } from './CardContent';
+import { CardHeader } from './CardHeader';
+import { WalletBackupType } from '../../../hooks/useWalletBackupPicker';
+
+interface WalletTypeCardProps {
+    type: WalletBackupType;
+    isSelected: boolean;
+    setSelectedType: (type: WalletBackupType) => void;
+    isVisible: boolean;
+}
+
+const containerStyle = prepareNativeStyle<{ isSelected: boolean }>((utils, { isSelected }) => ({
+    marginTop: utils.spacings.sp16,
+    borderColor: isSelected ? utils.colors.borderSecondary : 'transparent',
+    borderWidth: utils.borders.widths.large,
+}));
+
+const variantByWalletBackupType: Record<WalletBackupType, FullAlertProps['variant']> = {
+    'single-share': 'success',
+    'multi-share': 'warning',
+    '12-words': 'neutral',
+    '24-words': 'neutral',
+};
+
+export const WalletBackupCard = memo(
+    ({ type, isVisible, isSelected, setSelectedType }: WalletTypeCardProps) => {
+        const { applyStyle } = useNativeStyles();
+        const { translate } = useTranslate();
+        const openLink = useOpenLink();
+
+        const handleLearnMorePress = () => {
+            openLink(HELP_CENTER_MULTI_SHARE_BACKUP_URL);
+        };
+
+        const selectCard = () => {
+            setSelectedType(type);
+        };
+
+        if (!isVisible) return null;
+
+        return (
+            <Pressable onPress={selectCard}>
+                <Card style={applyStyle(containerStyle, { isSelected })}>
+                    <CardHeader isSelected={isSelected} type={type} />
+                    <CardDivider horizontalPadding={isSelected ? 'sp16' : 'sp24'} />
```

Comment:
Why is the padding necessary? When I remove it, the UI stays the same (tested on android).
```suggestion
                    <CardDivider />
```

Reply:
The reason is the `selected/unselected` state of the card. The divider shouldn’t overlap the border and if the styles are not applied the divider does not reach the edges. I don’t want to change the border width because it would cause an ugly jump when a user selects a card. We can huddle where I can explain with a demo
![1743762682815](https://github.com/user-attachments/assets/82ce2e84-4283-4fe0-9f05-716678ee4c39)

![Screenshot_2025-04-04-12-30-47-230_io trezor suite debug](https://github.com/user-attachments/assets/5c8b5766-7e14-48bc-bfbd-da079af6bd6f)

Reply:
![Screenshot 2025-04-07 at 9 36 22 AM](https://github.com/user-attachments/assets/24a6d54e-5de6-4ef0-ac9c-6d9d4063834e)

The divider is overflowing the card width. It is barely visible with these colors, but it is. For selected cards the  `horizontalPadding`  should be `sp16`, for unselected 18 I believe.

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/614ee4b4f118a0e31c642a2014823012d4b420ae hah, in the fixup I messed it up, thanks for pointing this out :D